www/include/monitoring/status/Common/commonJS.php

```diff
@@ -350,34 +350,34 @@ function construct_HostGroupSelectList(id) {
                                         WHERE hg.hostgroup_id = hhg.hostgroup_id 
                                         AND hg.name NOT LIKE 'meta\_%' 
                                      ORDER BY hg.name");
-while ($hostgroups = $DBRESULT->fetchRow()) {
-    if ($centreon->user->access->admin || ($centreon->user->access->admin == 0 && isset($hgBrk[$hostgroups["name"]]))) {
-        if (!isset($tabHG)) {
-            $tabHG = array();
-        }
-        if (!isset($tabHG[$hostgroups["name"]])) {
-            $tabHG[$hostgroups["name"]] = "";
-        } else {
-            $tabHG[$hostgroups["name"]] .= ",";
+        while ($hostgroups = $DBRESULT->fetchRow()) {
```

Comment:
wrong indentation -> the problem comes from line 348. And seems to impact all the lines until line 380